src/gazebo_mavlink_interface.cpp

```diff
@@ -957,6 +962,64 @@ void GazeboMavlinkInterface::IRLockCallback(IRLockPtr& irlock_message) {
   mavlink_interface_->send_mavlink_message(&msg);
 }
 
+void GazeboMavlinkInterface::ArucoMarkerCallback(ArucoMarkerPtr& arucoMarker_message) {
```

Comment:
Can we keep this a little more generic? e.g. targetReleativeCallback

Reply:
Yes, done in https://github.com/PX4/PX4-SITL_gazebo-classic/pull/950/commits/21f9a3e8e1e9b8b405da67146f57883d077ff9b0